src/gsi/crtm_interface.f90

```diff
@@ -555,10 +560,16 @@ subroutine init_crtm(init_pass,mype_diaghdr,mype,nchanl,nreal,isis,obstype,radmo
  if (radmod%laerosol_fwd) then 
     if(.not.allocated(aero)) allocate(aero(nsig,n_actual_aerosols))
     if(.not.allocated(aero_conc)) allocate(aero_conc(msig,n_actual_aerosols),auxrh(msig))
+    if(.not.allocated(aero_wc)) allocate(aero_wc(msig,n_actual_aerosols)) 
     n_actual_aerosols_wk=n_actual_aerosols
     n_aerosols_fwd_wk=n_aerosols_fwd
     n_aerosols_jac_wk=n_aerosols_jac
     Load_AerosolCoeff=.true.
+     if (fv3_cmaq_regional) then
```

Comment:
Seems Aerosol_Model, AerosolCoeff_File and AerosolCoeff_Format are not used in this programs 
and there are comments on lines like line 605
![image](https://user-images.githubusercontent.com/15043351/169887083-54e8edd6-86f9-456b-8a56-53495144dbe1.png)
Should they (including those comments)  be removed  or all are commented out but to be kept as they are to help for future development/documentation?

Reply:
The JCSDA will release a new version of CRTM to public that will support this capability. For now I would like to keep the lines there for my current RRFS-CMAQ/AOD assimilation with an internal version of CRTM from JCSDA.  This may be good for future development and documentation as you mentioned.

Reply:
Got it!  Accepted! Thanks.